docs/_interface/Management-APIs.md

```diff
@@ -1,1399 +0,0 @@
----
-title: Management APIs
```

Comment:
Same question again. Is this replaced by anything?

Reply:
This is replaced by the individual RPC namespace pages.